src/monitor.rs

```diff
@@ -219,7 +221,10 @@ macro_rules! monitor_cmd_for {
 					match res {
 						Ok(Some(Phase::Signed)) => Ok(()),
 						Ok(Some(_)) => Err(Error::IncorrectPhase),
-						Ok(None) => unreachable!("Phase should always exist"),
+						Ok(None) => {
+							log::warn!(target: LOG_TARGET, "EPM::Phase was not found at: {:?}", hash);
```

Comment:
I have seen this on my dev node when no signed phase occured yet.

Reply:
Seems like subxt is not capable of parsing the defaults that we define for storage types. 

https://github.com/paritytech/substrate/blob/49b06901eb65f2c61ff0934d66987fd955d5b8f5/frame/election-provider-multi-phase/src/lib.rs#L1193

The `default` for `Phase` is `None`. You can just hardcode this for now.

Reply:
Similarly, the default for `Round` is 1:

https://github.com/paritytech/substrate/blob/49b06901eb65f2c61ff0934d66987fd955d5b8f5/frame/election-provider-multi-phase/src/lib.rs#L1188